src/__tests__/Search.test.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { render, screen, fireEvent } from '@testing-library/react';
+import { MemoryRouter } from 'react-router-dom';
+import AnimeSearchPage from '../components/SearchPage';
+
+describe('AnimeSearchPage component', () => {
+  it('should show search results when the search button is clicked', async () => {
+    render(
+      <MemoryRouter>
+        <AnimeSearchPage />
+      </MemoryRouter>,
+    );
+
+    const searchInput = screen.getByPlaceholderText(/search for anime/i);
+    const searchButton = screen.getByRole('button', { name: /search/i });
+
+    fireEvent.change(searchInput, { target: { value: 'Naruto' } });
+    fireEvent.click(searchButton);
+
+    // Wait for API request and search results to update
+    await screen.findByText((content, element) => {
+      const normalizedText = content.toLowerCase();
+      const elementText = element.textContent.toLowerCase();
+      return elementText.includes('naruto') && normalizedText === 'naruto';
+    });
+
```

Comment:
- [ ] Dear @MoyasiGinko, Just as mentioned by the previous reviewer One of your test is falling because it fails to find an element on the document, Please use my screenshot to debug the issues and make the appropriate changes to be able to pass all the tests. 

### Console output 
![image](https://github.com/MoyasiGinko/React-Capstone/assets/69077061/5c24e69b-08e7-483c-8a18-0ab01780a4f9)

Reply:
Hey @TadesseDev , i have tried the same test on my terminal it works fine, i dont know what is wrong. i will provide you a screenshot please see below,

![image](https://github.com/MoyasiGinko/React-Capstone/assets/108092696/1b9614ce-d1bb-4a33-963e-8e4423af9e26)

Reply:
dear @MoyasiGinko, This is usually why we need to test, Just because it works on the developer's computer doesn't mean it works everywhere. We need to make sure tests are passing in all the test machines, Please use a different/new computer, clone the project, and run the test.